### DISCUSSION TITLE
Show a default plan with 2TB storage when running the server locally

### DISCUSSION THREAD

**Discussion by mnvr** [ente-io/ente#729] [Enhancements]
Right now, when we create an account in a locally running server, it shows the following default plan selector

<img width="457" alt="Screenshot 2024-03-07 at 09 56 41" src="https://github.com/ente-io/ente/assets/24503581/35ca9894-a0bb-4271-aebb-757d5a099f15">

This is working as expected, because when running locally there are no paid plans present. But for convenience it is better show a much larger plan in these circumstances, or expand the scope of the trial (say 2T, 20 years) to reduce the amount of fiddling self hosters need to do.

**Comment by mnvr** [comment]
This is now done. You can use `ente admin update-subscription` CLI command to increase the storage quota and account validity for users on your self hosted instance. For more details, see: https://help.ente.io/self-hosting/guides/admin.

The CLI also provides a bunch of other commands that you might useful for administering your instance. However, and just for completeness' sake (using the CLI is less error-prone and more convenient), if you wish to allocate extra space on your self hosted instance without using the CLI, you could've also run the following SQL query against the database for your self hosted instance:

```sql
INSERT INTO storage_bonus (bonus_id, user_id, storage, type, valid_till) VALUES ('self-hosted-myself', (SELECT user_id FROM users LIMIT 1), 1099511627776, 'ADD_ON_SUPPORT', 0)
```

**Reply by nprime496** [reply]
Hi, the page https://help.ente.io/self-hosting/guides/admin is now unavailable, please where is the latest documentation?

**Reply by vishnukvmd** [reply]
Hi, this has been moved here: https://help.ente.io/self-hosting/administration/cli#step-4-increase-storage-and-account-validity

**Reply by woelper** [reply]
Leaving this here for others and as feedback:

The CLI can be super hard to install ([see here](https://github.com/ente-io/ente/issues/722)), especially on minimal systems as it requires gnome-keyring. This leaves SQL as the only option.

On docker setups, users will need to do something like `docker exec -it my-ente-postgres-1 psql -U pguser -d ente_db` to be able to interact with SQL.

**Comment by Patten310** [comment]
Hi,
i dont get it work on my raspi local Installation. I have added the ID to the admin section. But the command
admin update-subscription -a mail -u mail --no-limit dont change anything. Thanks for your help
